### ISSUE TITLE
Embedded subtitles overrules external

### ISSUE THREAD

**Issue by GoogleCodeExporter** [0xfab-ri/universal-media-server#35]
```
What steps will reproduce the problem?
1. Play a mkv with srt in one lnaguage, and embedded with another

What is the expected output? What do you see instead?
I think, if the user has done the effort to put a srt file, then it's because 
the embedded subtitles are suitable for the user, and the srt should be used. 

What version of the product are you using? On what operating system?
2.2.5


```

Original issue reported on code.google.com by `jakobste...@gmail.com` on 17 Jan 2013 at 8:45

**Comment by GoogleCodeExporter** [comment]
```
gahh..i for the word NOT in this sentence:

"if the user has done the effort to put a srt file, then it's because the 
embedded subtitles are >NOT< suitable for the user, and the srt should be used" 


```

Original comment by `jakobste...@gmail.com` on 17 Jan 2013 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
This wasn't a bug but had to do with the default language options in the 
program. Anyway I have put the default option back now so it will choose the 
external one first. 
```

Original comment by `subjunk` on 9 Feb 2013 at 9:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
But there still seesm to be an issue with some renders
http://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=623

When play the one file that doesent load the SRT by default with WD TV Live, 
but it plays fine with my Bravia (its getting transcoded with mencoder/subs on 
both)
```

Original comment by `jakobste...@gmail.com` on 9 Feb 2013 at 9:44

**Comment by GoogleCodeExporter** [comment]
```
The fixes will be in 2.4.0 so if that bug is still around when you've tried 
that version (which I'm releasing today), reply to that thread and we can work 
on it
```

Original comment by `subjunk` on 9 Feb 2013 at 9:54

**Comment by GoogleCodeExporter** [comment]
```
Sounds great, thanks!
```

Original comment by `jakobste...@gmail.com` on 9 Feb 2013 at 9:56
